tests/test_ddebug_canc/test.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+make clean
```

Comment:
This commit shouldn't be in this pull request.
But it's needed addition to verificarlo.